src/Tizen.Multimedia/AudioManager/AudioStreamPolicy.cs

```diff
@@ -304,6 +306,118 @@ public void RemoveDeviceForStreamRouting(AudioDevice device)
                 ThrowIfError("Failed to remove device for stream routing");
         }
 
+        /// <summary>
+        /// Gets or sets the preferred input device.
+        /// </summary>
+        /// <value>
+        /// The <see cref="AudioDevice"/> instance.<br/>
+        /// The default is null which means any device is not set on this property.
+        /// </value>
+        /// <remarks>
+        /// This property is to set a specific built-in device when the system has multiple devices of the same built-in device type.
+        /// When there's only one device for a built-in device type in the system, nothing will happen even if this property is set successfully.
+        /// </remarks>
+        /// <exception cref="ArgumentException">A device is not for input.</exception>
+        /// <exception cref="InvalidOperationException">An internal error occurs.</exception>
+        /// <exception cref="AudioPolicyException">A device is not supported by this <see cref="AudioStreamPolicy"/> instance.</exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="AudioStreamPolicy"/> has already been disposed of.</exception>
+        /// <seealso cref="AudioManager.GetConnectedDevices()"/>
+        /// <since_tizen> 6 </since_tizen>
+        public AudioDevice PreferredInputDevice
+        {
+            get
+            {
+                if (_inputDevice == null)
+                {
+                    return null;
+                }
+
+                Interop.AudioStreamPolicy.GetPreferredDevice(Handle, out var inDeviceId, out _).
+                    ThrowIfError("Failed to get preferred input device");
+
+                if (inDeviceId != _inputDevice.Id)
+                {
+                    throw new InvalidOperationException("output device id does not match");
+                }
+
+                return _inputDevice;
+            }
+            set
+            {
+                int deviceId;
+
+                if (value == null)
+                {
+                    deviceId = 0;
+                }
+                else
+                {
+                    deviceId = value.Id;
+                }
+
+                Interop.AudioStreamPolicy.SetPreferredDevice(Handle, AudioDeviceIoDirection.Input, deviceId).
```

Comment:
How about the following code to simplify all above code.

```csharp
Interop.AudioStreamPolicy.SetPreferredDevice(Handle, AudioDeviceIoDirection.Input, value?.Id ?? 0).
      ThrowIfError("Failed to set preferred input device");
```

Reply:
Okay, it looks good. I'll do that.